tests/integration/delete_user.t

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/perl -w
+
+use ProductOpener::PerlStandards;
+
+use Test::More;
+use ProductOpener::APITest qw/:all/;
+use ProductOpener::Test qw/:all/;
+use ProductOpener::TestDefaults qw/:all/;
+use ProductOpener::Users qw/:all/;
+use ProductOpener::Producers qw/:all/;
+
+remove_all_users();
+wait_application_ready();
+
+#new common user agent
+my $ua = new_client();
+my %create_client_args = (%default_user_form, (email => 'bob@test.com'));
+create_user($ua, \%create_client_args);
+
+#new admin user agent
+my $admin = new_client();
+create_user($admin, \%admin_user_form);
+
+#common ua add a new product then delete the account while being still logged in
+edit_product($ua, \%default_product);
+
+my @words = (
+	"User is being deleted. This may take a few minutes.",
+	"See you soon!",
+	"Invalid user.",
+	"Unknown user.",
+	"Incorrect user name or password.",
+	"/editor/anonymous"
+);
```

Comment:
We can remove it now I think.
```suggestion
```

Reply:
Oups, after all these commits I even forgot to remove this